test-executor/pom.xml

```diff
@@ -123,6 +123,7 @@
 
   <build>
     <plugins>
+      <!-- ToDo: remove me when all felix dependencies are removed -->
```

Comment:
Please clear also dependencies:
```
   <dependency>
      <groupId>org.apache.sling</groupId>
      <artifactId>org.apache.sling.commons.osgi</artifactId>
    </dependency>
```
and
```
    <dependency>
      <groupId>org.apache.felix</groupId>
      <artifactId>org.apache.felix.scr.annotations</artifactId>
    </dependency>
```

Reply:
I edited wrong file, now should be correct